resources/monitoring/README.md

```diff
@@ -8,6 +8,24 @@ This chart installs the Prometheus operator along with [Grafana](https://grafana
 
 All the monitoring components run in the `kyma-system` Namespace.
 
+## Custom Resource Definitions
+
+Custom resource definitions for `crd-alertmanager.yaml`, `crd-podmonitor.yaml`, and `crd-servicemonitor.yaml` have been moved to the `resources/cluster-essentials/templates` folder and will be maintained there. Along with the next update of the Prometheus Operator, update the listed files in the [Cluster Essentials](https://github.com/kyma-project/kyma/tree/master/resources/cluster-essentials) chart.
+
+## Kyma Customizations
```

Comment:
```suggestion
## Kyma customizations
```
I changed it because we usually capitalize just the first word in the title (unless it's a name)